Tools/CISupport/ews-build/steps.py

```diff
@@ -236,6 +236,20 @@ def query_graph_ql(self, payload):
             data = json.loads(response.text)
             defer.returnValue(data)
 
+    @defer.inlineCallbacks
+    def get_number_of_prs_with_label(self, label='\"safe-merge-queue\"'):
```

Comment:
This shouldn't have a default argument, since we only every call it by passing an argument (and I can't really see how future code would benefit from a default argument)